src/profile_manager/forms.py

```diff
@@ -28,9 +28,12 @@ def __init__(self, *args, **kwargs):
 
         super().__init__(*args, **kwargs)
 
-        self.fields['grad_year'] = forms.ChoiceField(
-            choices=Profile.get_grad_year_choices()
-        )
+        if SiteConfig.get().custom_profile_field:
```

Comment:
Can you ensure this doesn't also trigger in silly cases, like if there's a space character in the field? I can't recall if a space character is considered False in python.

Reply:
a space character is considered True. i'll add a clean function that strips beginning and ending spaces